### PR TITLE
Bundle node

### PR DESCRIPTION
**Pull request by Brocco**

### CODE REVIEW

File: .github/workflows/composite/build/action.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+name: 'Verify'
+description: 'Verifies the current branch'
+
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/setup-node@v4
+      with:
+        node-version: '22.x'
```
Comment (marco-ippolito) [inline .github/workflows/composite/build/action.yml:9]: is there a specific reason for Node 22? or we can set to latest

File: apps/cli/src/lib/ensure-version.ts [hunk 2]
```diff
@@ -1,24 +1,51 @@
...
-    }
-  );
+  const url = `https://registry.npmjs.org/${pkgName}`;
+  return new Promise((resolve, reject) => {
+    const options = {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+    const request = https.get(url, options, (response) => {
```
Comment (marco-ippolito) [inline apps/cli/src/lib/ensure-version.ts:15]: can we use fetch instead of `https.get`?

Comment (Brocco) [reply apps/cli/src/lib/ensure-version.ts:15]: I was concerned about using fetch because this needs to run in node v14+ and IIRC fetch was added after 14

Comment (marco-ippolito) [reply apps/cli/src/lib/ensure-version.ts:15]: yes thats what I thought too, just wanted to confirm, fetch has been marked stable from 18

File: apps/cli/src/lib/ensure-version.ts [hunk 3]
```diff
@@ -1,24 +1,51 @@
...
+      const chunks = [] as string[];
```
Comment (JessicaSachs) [inline apps/cli/src/lib/ensure-version.ts:16]: `as` forces the type.

```suggestion
      const chunks: string[] = [];
```

**Review by JessicaSachs — APPROVED** [review]
Mini nit

**Review by JessicaSachs — APPROVED** [review]

**Pull request merged**